### PR TITLE
#9 Fix GROUP BY <integer>

### LINKED ISSUE #9: VS Workaround for `GROUP BY <integer>`
## Summary

When users combine integer constants as expressions with `DISTINCT`, the engine pushes down `GROUP BY <integer-constant>`. Exasol interprets this as "group by column number", which yields wrong results. What should happen here is that you get a result with a single group. `GROUP BY TRUE`, `GROUP BY '0'` or any other non-integer constant will work as expected.

Note that some databases interpret this as "group by integer constant". So the behavior is database specific. The reason is that this case is *not covered by the SQL standard*.

Workaround:

The adapter should work around this by turning `GROUP BY <integer-constant>` into `GROUP BY <other-constant>`.

Theoretically that should be done on a db-by-db basis, but that is too much effort, so a general workaround in VSCL and VSCJ should be implemented. This can be overridden in the dialect if there is ever the need.

## Features

- Turns `GROUP BY <integer-constant>` into `GROUP BY <other-constant>`

### PR DESCRIPTION
**Pull request by kaklakariada**
Closes #9

### CODE REVIEW

File: .busted [hunk 1]
```diff
@@ -2,17 +2,19 @@ return {
...
     shuffle = true,
-    lpath = "./src/?.lua;./spec/?.lua"
+    recursive = true,
+    verbose = true,
+    lazy = false,
+    ["auto-insulate"] = true,
+    ["keep-going"] = true,
+    ["defer-print"] = false,
+    lpath = "./src/?.lua;./spec/?.lua",
+    ROOT = {"spec/"},
```
Comment (ckunki) [inline .busted:12]: Please explain to me what the "busted" file means.

File: .lua-format [hunk 2]
```diff
@@ -0,0 +1,33 @@
+# https://github.com/Koihik/LuaFormatter/blob/master/docs/Style-Config.md
```
Comment (ckunki) [inline .lua-format:1]: This could be a candidate for generation by pk, right?

**Review by ckunki — DISMISSED** [review]
looks OK to me, though not an expert.
Some questions not directly related to the review.

File: .lua-format (hunk 2 above)
Comment (kaklakariada) [reply .lua-format:1]: Yes, correct!

File: .busted (hunk 1 above)
Comment (kaklakariada) [reply .busted:12]: Busted is a test runner for Lua, this is it's config file.

**Review by ckunki — APPROVED** [review]

**Pull request merged**